002-stem-probes/README.md

```diff
@@ -0,0 +1,35 @@
+# Georgios Varnavides: UC Berkeley Postdoctoral Miller Research Fellow
+
+_August 15, 2024_
+
+[Georgios Varnavides](https://georgiosvarnavides.com/) is a postdoctoral Miller research fellow at UC Berkeley, where he works on material science. He's a marimo power user who cares deeply about digital science communication. 👨‍🔬🎙️
+
+Georgios has been using [marimo's WebAssembly features](https://docs.marimo.io/guides/wasm.html) to create interactive science content that runs directly in the browser. His work spans from notebooks to apps and even interactive slides. One of his notable contributions is an interactive presentation on STEM probes, which you can explore here:
+
+[![Open with marimo](https://marimo.io/shield.svg)](https://marimo.io/p/@gvarnavides/stem-probes)
+
+## Innovative Combinations
+
+Georgios has taken his work a step further by finding a way to combine Observable Framework and marimo, utilizing an advanced marimo feature called [marimo islands](https://docs.marimo.io/guides/exporting.html#embed-marimo-outputs-in-html-using-islands) 🏝️. This innovative approach showcases the flexibility and power of marimo for creating interactive scientific content.
+
+## Local Exploration
+
+You can explore Georgios' notebook locally with:
+
+```shell
+uvx marimo edit --sandbox stem-probes.py
+```
+
+if you have `uv` installed, or
+
+```shell
+marimo run --sandbox 002-stem-probes\stem-probes.py
```

Comment:
Should we recommend `marimo edit` as a general rule?

Additionally, let's use POSIX-style paths with forward slashes.

```suggestion
marimo edit --sandbox 002-stem-probes/stem-probes.py
```

Reply:
I was pondering on whether to use `edit` or `run`; as these notebooks are meant to be run; they are finished programs showcasing various implementations using marimo.

Sure, will change it to `edit` as the user can always toggle apple view to see the outputs.

Reply:
@Haleshot , that makes sense. My concern was that in the past users saw `marimo run` and didn't know how to edit the notebook (they thought they had to edit the .py file directly). We can recommend both.